iothub/service/tests/ConnectionString/ServiceClientConnectionStringTests.cs

```diff
@@ -26,7 +26,7 @@ public virtual IotHubConnectionStringBuilder Populate(IotHubConnectionStringBuil
         public void ServiceClientConnectionStringDefaultScopeDefaultCredentialTypeTest()
         {
             string connectionString = "HostName=acme.azure-devices.net;SharedAccessKeyName=AllAccessKey;SharedAccessKey=dGVzdFN0cmluZzE=";
-            var serviceClient = (AmqpServiceClient)ServiceClient.CreateFromConnectionString(connectionString);
+            var serviceClient = (ServiceClient)ServiceClient.CreateFromConnectionString(connectionString);
```

Comment:
Does it still require casting now?

Reply:
Good point. It should not. I will double check and update accordingly.